apps/api/src/app/workflows-v2/dtos/controls/delay-control.dto.ts

```diff
@@ -1,30 +1,31 @@
-import { ApiProperty } from '@nestjs/swagger';
-import { DelayTypeEnum, TimeUnitEnum } from '@novu/shared';
-import { IsEnum, IsNumber, Min, ValidateIf } from 'class-validator';
+import { ApiPropertyOptional } from '@nestjs/swagger';
+import { TimeUnitEnum } from '@novu/shared';
+import { IsEnum, IsNumber, IsOptional, IsString, Min, MinLength } from 'class-validator';
 import { SkipControlDto } from './skip.dto';
 
 export class DelayControlDto extends SkipControlDto {
-  @ApiProperty({
-    description: "Type of the delay. Currently only 'regular' is supported by the schema.",
-    enum: [DelayTypeEnum.REGULAR],
-    default: DelayTypeEnum.REGULAR,
-  })
-  @IsEnum({ REGULAR: DelayTypeEnum.REGULAR })
-  type: DelayTypeEnum.REGULAR;
```

Comment:
the `type` field is not used at all on the delay step, same about the digest it is not even saved in the control values;
but this change requires removing the field from the old delay `uiSchema`

Reply:
@LetItRock can we still keep it? as cron and amount & unit can't really work together. And I also need the type for the dynamic key approach.